src/common/messages/PeerUnavailable.py

```diff
@@ -0,0 +1,17 @@
+"""Peer unavailable notification."""
```

Comment:
Czy nie byłoby czytelniej zrobić pliku z `messages.py` ze wszystkimi poniższymi klasami? :D 
Chyba czytelniej by się to czytało :)